pycyphal/_version.py

```diff
@@ -1 +1 @@
-__version__ = "1.19.0"
+__version__ = "1.20.0"
```

Comment:
Sorry, since this is a fix, can you please bump the [patch](https://semver.org/) version instead?